README.md

```diff
@@ -29,6 +29,20 @@ The steps for contributing changes are the following,
 
 Please make sure your code is well organized and commented before opening the pull request.
 
+## PR labeling strategy
+
+On a daily basis, all PRs should be labeled with one of the review process tags:
+- `PR: Draft`
+- `PR: Needs Review`
+- `PR: Work Needed`
+- `PR: Mergeable`
+
+The person opening the PR should apply `PR: Draft` until the work is finished, then the person can apply the `PR: Needs Review` label as soon the PR is ready to be reviewed.
```

Comment:
Is it up to the person drafting the PR to assign a reviewer at the point he/she adds the label 'PR: Needs Review'?

Reply:
@Janetbananet I rephrased this sentence to be more clear about responsibilities. In my opinion, the person that works on the PR can assign someone else to review. If she/he don't know who to assign the PR to, she/he should let the team know on the Slack channel and we can follow up from there.